src/ClapCommandLineHandler.package/ClapCommandLineHandler.class/class/hasCommand.st

```diff
@@ -0,0 +1,5 @@
+testing
+hasCommand
+"This method checks if the subClasses have a method 'command'."
+	self class methodDictionary at: #command ifPresent: [ ^ true ] .
```

Comment:
Oh and this really means that `command` must be implemented in concrete handlers… so why not make it abstract:
```smalltalk
ClapCommandLineHandler >> command
    ^ self subclassResponsibility
```

Granted, this might be a problem if a subclass forgets to implement `command` and inherits the abstract definition, because that will fail when the command is invoked, but the current code has a similar edge case. This comforts me about the approach with pragma + merging into an anonymous root command.